examples/embedded-graphics-wii/src/main.rs

```diff
@@ -22,11 +22,10 @@ fn main(_argc: isize, _argv: *const *const u8) -> isize {
     Input::init(ControllerType::Gamecube);
     Input::init(ControllerType::Wii);
 
-    let mut gcn_ctrl = Input::new(ControllerType::Gamecube, ControllerPort::One);
-    let mut wii_ctrl = Input::new(ControllerType::Wii, ControllerPort::One);
-    wii_ctrl.set_data_fmt(DataFmt::ButtonsAccelIR);
+    let gcn_ctrl = Input::new(ControllerPort::One, ControllerType::Gamecube);
```

Comment:
Why were the parameters re-arranged?

Reply:
I dont actively have a reason. Its much easier to read to me I guess.

Reply:
I agree. I was just curious.

Reply:
I don't really care - tbh I prefer the ControllerPort, ControllerType setup.